cmd/agent/install_mac_os.sh

```diff
@@ -423,7 +435,6 @@ else
     # the GUI was not running for the user (e.g. a run of this script via
     # ssh for user not logged in via GUI).
     if $cmd_launchctl print "gui/$user_uid/$service_name" 1>/dev/null 2>/dev/null; then
```

Comment:
AFAICS we're now doing all of the below commands in the preinst script, so why remove just one of them? Let's either remove all or leave them all (I think leaving them all is preferable because it ensures backwards compatibility)

Reply:
The issue is that in `system-wide` it seems that we always have an agent started first in single-user mode and then is always stopped and unloaded. Hence why I added the `tray-not-started` file previously. Everytime we start the install script in system-wide we are going into this condition so the way to keep it retro-compatible was to only run the osascript with older versions which won't have the `tray-not-started` file. 

I moved it in the prescript to not use the `tray-not-started` file as an alternative.